defender-endpoint/mde-plugin-wsl.md

```diff
@@ -293,6 +293,8 @@ DeviceProcessEvents
          dnsProxy=false
          ```
 
+   - If the connectivity issues persist, please collect the networking logs using the [Steps](https://github.com/microsoft/WSL/blob/master/CONTRIBUTING.md#8-collect-wsl-logs-recommended-method) and send along with support bundle.
```

Comment:
use the link as https://aka.ms/wsllogs.

If the connectivity issues persist, please collect the logs using the method mentioned at <link> and send along with support bundle.